src/libs/state.ts

```diff
@@ -111,6 +115,8 @@ export const authStore: Atom<AuthState> = atom<AuthState>({
 
 export const getUser = (): TerraUser => authStore.get().user;
 
+export const getOidcUser = (): OidcUser | undefined => authStore.get().oidcUser;
```

Comment:
Is this ever used?

Generally, I think it would be preferable to avoid proliferation of things that reach into global state. So if it's not needed, I'd rather not add it.

Reply:
It was in a previous commit, but it is now longer used. I'll remove it for now.